From e0d85e9a33e83e2166f39ede5336c2b9e640ff1e Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Wed, 19 Nov 2014 11:22:53 +0000
Subject: [PATCH 20/23] Fix typo in the name of the info entry containing
 uptime

I'm assuming this is a typo, although it's been this way since it was added in r280

Signed-off-by: Jon Turney <jon.turney@dronecode.org.uk>
---
 src/client/windows/crash_generation/client_info.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/windows/crash_generation/client_info.cc b/src/client/windows/crash_generation/client_info.cc
index 30a48db..fb29362 100644
--- a/src/client/windows/crash_generation/client_info.cc
+++ b/src/client/windows/crash_generation/client_info.cc
@@ -30,7 +30,7 @@
 #include "client/windows/crash_generation/client_info.h"
 #include "client/windows/common/ipc_protocol.h"
 
-static const wchar_t kCustomInfoProcessUptimeName[] = L"ptime";
+static const wchar_t kCustomInfoProcessUptimeName[] = L"uptime";
 static const size_t kMaxCustomInfoEntries = 4096;
 
 namespace google_breakpad {
-- 
2.7.0

